src/main/java/seedu/address/logic/commands/AddTuitionTaskCommand.java

```diff
@@ -19,67 +20,75 @@
  */
 public class AddTuitionTaskCommand extends UndoableCommand {
 
-    public static final String COMMAND_WORD = "addTuition";
+    public static final String COMMAND_WORD = "addtuition";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a tuition (task) into the schedule.\n"
             + "Parameters: "
-            + "last shown index of person associated with tuition" + "(space) "
+            + "tutee_index"
             + "Date(dd/mm/yyyy) "
             + "Start time(hh:mm) "
             + "Duration(XXhXXm) "
-            + "Description.\n"
+            + "Description( anything; leading and trailing whitepsaces will be trimmed )\n"
             + "Example: " + COMMAND_WORD + " "
             + "1 "
             + "10/12/2018 "
             + "12:30 "
             + "1h30m "
-            + "Calculus homework page 24.";
+            + "Calculus homework page 24";
 
     public static final String MESSAGE_SUCCESS = "New tuition task added.";
 
-    private final TuitionTask toAdd;
     private final Index targetIndex;
+    private final LocalDateTime taskdateTime;
+    private final String duration;
+    private final String description;
 
+    private TuitionTask toAdd;
     private TuitionSchedule tuitionSchedule;
-    private Tutee associatedPerson;
+    private Tutee associatedTutee;
 
     /**
-     * Creates an AddTuition to add the specified {@code Task} which is associated to {@code Person}.
+     * Creates an AddTuition to add the specified {@code Task} which is associated to {@code Tutee}.
      */
-    public AddTuitionTaskCommand(TuitionTask task, Index indexOfAssociatedPerson) {
-        requireNonNull(task);
-        toAdd = task;
-        targetIndex = indexOfAssociatedPerson;
+    public AddTuitionTaskCommand(Index targetIndex, LocalDateTime taskDateTime, String duration, String description) {
+        requireNonNull(taskDateTime);
+        requireNonNull(duration);
+        requireNonNull(description);
+        this.taskdateTime = taskDateTime;
+        this.duration = duration;
+        this.description = description;
+        this.targetIndex = targetIndex;
     }
 
     @Override
     public CommandResult executeUndoableCommand() {
-        requireNonNull(tuitionSchedule);
         tuitionSchedule.addTask(toAdd);
         model.addTask(toAdd);
         return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd));
     }
 
     @Override
     protected void preprocessUndoableCommand() throws CommandException {
-        findAssociatedPerson();
-        assert(associatedPerson.getTuitionSchedule() != null);
-        tuitionSchedule = associatedPerson.getTuitionSchedule();
+        associatedTutee = getAssociatedTutee();
+        requireNonNull(associatedTutee.getTuitionSchedule());
+        tuitionSchedule = associatedTutee.getTuitionSchedule();
+        toAdd = new TuitionTask(associatedTutee, taskdateTime, duration, description);
     }
```

Comment:
tuition task is created here after the associated tutee is found using the given index